NotesBackEnd/tests/Notes.Api.Tests/Controllers/NotesControllerTest.cs

```diff
@@ -113,15 +107,22 @@ public void Init()
         {
             var expectedNote = new Note { Text = "Fourth note", Id = new Guid("4785546e-824d-42a4-900b-e656f19ffb59") };
 
-            IHttpActionResult response = await _controller.DeleteAsync(Guid.NewGuid());
-            HttpResponseMessage executedResponse = await response.ExecuteAsync(CancellationToken.None);
-            executedResponse.TryGetContentValue(out Note actualNote);
+            var (actualNote, responseMessage) = await GetExecutedResponse<Note>(() => _controller.DeleteAsync(Guid.NewGuid()));
 
             Assert.Multiple(() =>
             {
-                Assert.That(executedResponse.StatusCode, Is.EqualTo(HttpStatusCode.OK));
+                Assert.That(responseMessage.StatusCode, Is.EqualTo(HttpStatusCode.OK));
                 Assert.That(actualNote, Is.EqualTo(expectedNote).UsingNoteComparer());
             });
         }
+
+        internal async Task<(T ActualContent, HttpResponseMessage ResponseMessage)> GetExecutedResponse<T>(Func<Task<IHttpActionResult>> controllerFunction)

```

Comment:
Really nice :-)
This method should be `private` tough, as it is used in this controller only.